src/controllers/api.controller.ts

```diff
@@ -480,8 +481,15 @@ export const claimReport = async (
     // those can be separate PRs lets start with this one working and the design for it
     //
 
+    const edge = await prisma.edge.findFirst({
```

Comment:
what is this and why?  I don't understand it can you comment, how did it work before?

Reply:
also above there can only be one result by id, id is a primary key for Claim